File: app/src/main/java/ir/mymessage/view/interfaces/DialogsInterface.java
package ir.mymessage.view.interfaces;

import android.content.Context;

import java.util.ArrayList;

import ir.mymessage.model.local.DialogLocal;

public interface DialogsInterface {
    void setupDialogsActivity();
    void startFriendsActivity();
    void startMessagesActivity(DialogLocal dialogLocal);
    void displayDialogs(ArrayList<DialogLocal> dialogArrayList);
    void showProgress();
    void hideProgress();
    Context getContext();
}

File: app/src/main/java/ir/mymessage/view/interfaces/DialogsInterface.kt
package ir.mymessage.view.interfaces

import android.content.Context

import java.util.ArrayList

import ir.mymessage.model.local.DialogLocal

interface DialogsInterface {
    fun getContext(): Context
    fun setupDialogsActivity()
    fun startFriendsActivity()
    fun startMessagesActivity(dialogLocal: DialogLocal)
    fun displayDialogs(dialogArrayList: ArrayList<DialogLocal>)
    fun showProgress()
    fun hideProgress()
}
